### ISSUE TITLE
Backup only works with DEBUG_MODE=TRUE in 3.5.2

### ISSUE THREAD

**Issue by maltesa** [tiredofit/docker-db-backup#176]
### Summary
The issue occurred after upgrading to the latest image `3.5.2`. Reverting to `3.4.2` resolves the issue.
The following backups were run subsequently and resulted in these files being uploaded to S3 (see filesizes).

#### Without `DEBUG_MODE=TRUE`
![without-debug](https://user-images.githubusercontent.com/4067078/195275796-dd96badb-ab9c-4849-a3d4-8b1e395c5a9a.png)

#### With `DEBUG_MODE=TRUE`
![with-debug](https://user-images.githubusercontent.com/4067078/195275876-91da8abb-7d81-4a57-8cdd-4c744f386cd9.png)

### Steps to reproduce
1. Run docker-compose (see environment)
2. Run backup

### What is the expected *correct* behavior?

Actual backups being uploaded into S3 Bucket WITHOUT `DEBUG_MODE=TRUE`

### Environment

```yaml
# docker-compose.yml
version: "3"

services:
  db-backup:
    image: tiredofit/db-backup:3.5.2
    environment:
      - BACKUP_LOCATION=S3
      - S3_HOST=eu2.contabostorage.com
      - S3_REGION=eu2
      - S3_BUCKET=backups
      - S3_PATH=postgres
      - S3_KEY_ID=abc
      - S3_KEY_SECRET=abc
      - CHECKSUM=SHA1
      - COMPRESSION=ZSTD
      - SPLIT_DB=TRUE
      - CONTAINER_ENABLE_MONITORING=FALSE
      - DEBUG_MODE=TRUE
      - DB_DUMP_BEGIN=0315 # begin at 03:15 o'clock
      - DB_DUMP_FREQ=1440 # every 24 hours
      - DB_CLEANUP_TIME=8640 # cleanup backups older than 1 week
      - DB_TYPE=pgsql
      - DB_HOST=postgres
      - DB_NAME=ALL
      - DB_USER=postgres
      - DB_PASS=wonttell
      - 
  postgres:
    image: postgres:14.5
    environment:
      POSTGRES_PASSWORD: wonttell
```

- Image version / tag:  `3.5.2`
- Host OS: Docker

### Possible fixes
Revert to `3.4.2` (haven't checked if `3.5.0` works)

**Comment by tiredofit** [comment]
Can you share some of the DEBUG_MODE=TRUE output specifically a couple of the `pg_dump -h` command ? I'm also curious if instead of `DEBUG_MODE=TRUE` you turn that off and switch to `SHOW_OUTPUT=TRUE` or `CONTAINER_LOG_LEVEL=DEBUG` ..

I'm trying to see if I've done anything to silence the postgres backups since 3.4.2 but not coming up with anything.

**Comment by tiredofit** [comment]
Never mind, I see it now. Going to revert those changes.

**Comment by tiredofit** [comment]
Resolved in `3.5.3.` building as we speak! https://github.com/tiredofit/docker-db-backup/commit/7e5e9d308dcc6b4d1019cd2994ae695ac89b06e7

**Comment by maltesa** [comment]
Ahh, awesome 🥳

I just replicated the issue locally and **can confirm that it is resolved with `3.5.3`**!

Thank you for your awesome work!

**Comment by tiredofit** [comment]
Great to hear! Thanks again.
